templates/lxc-local.in

```diff
@@ -180,6 +180,12 @@ process_excludes() {
   fi
 }
 
+prepare_rootfs() {
```

Comment:
Ok, so that's when that's introduced. You need to re-order the commits so this is added prior to the commit using it (important for bisect).

Also, any reason why this is called from unpack_rootfs rather than be called from the main code like the rest of the functions?

Reply:
I thought it made sense to call it from there, so the preparation function would automatically be called as part of the rootfs unpacking process.